augur/filter.py

```diff
@@ -1553,9 +1553,13 @@ def run(args):
         subsampled_strains = set()
         for group, queue in queues_by_group.items():
             records = []
+            # Each record is a pandas.Series instance.
             for record in queue.get_items():
-                # Each record is a pandas.Series instance. Track the name of the
-                # record, so we can output its sequences later.
+                # Skip if it has already been written to outputs.
+                if record.name in force_included_strains:
+                    continue
```

Comment:
I don't know this code or the desired behaviour of `augur filter` subsampling very well, so this might be a naive question, but: is it valid to reduce the size of the priority queue by the number of force-included records it happens to contain? Would it be better to exclude from subsampling any records which are force-included, and thus avoid duplicates that way instead?

Reply:
Valid point! afa9ef816c17e37680c250214489eb0e1d8716df should be the better approach.